week-1/K-functions-parameters/exercise4.js

```diff
@@ -1,5 +1,7 @@
-// Declare your function first
+function add(num1,num2){
+    return num1 + num2;
+}
+var sum = add (13, 124);
```

Comment:
Good, JS is not whitespace sensitive however for style practices having a space isn't between the function name and parentheses isn't recommended